package.json

```diff
@@ -9,10 +9,16 @@
     "url": "git://github.com/woocommerce/woocommerce-gateway-payfast.git"
   },
   "devDependencies": {
+    "@woocommerce/dependency-extraction-webpack-plugin": "^1.4.0",
+    "@woocommerce/eslint-plugin": "^1.1.0",
+    "@wordpress/scripts": "^13.0.2",
     "node-wp-i18n": "~1.2.3"
   },
```

Comment:
Should there be any `engines` definition in the `package.json` for node and npm?

Reply:
Added in b807907c93cb66b5e7698f0c7619275e31f7779c. I used the [requirements from `@wordpress/scripts`](https://github.com/WordPress/gutenberg/blob/trunk/packages/scripts/package.json#L21-L24) as a reference, but I see in the [Stripe PR you opened](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1467/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R82) npm's version is set to >6.0 instead of >6.9. Is it expected that they are different? I'm a bit confused by this. :confused:

Reply:
Stripe PR should probably be 6.9, what you did here is good.

Reply:
I just updated the Stripe PR engine definition to correct - good catch :)